detection-rules/impersonation_wise_suspicious_attachment.yml

```diff
@@ -0,0 +1,29 @@
+name: "Brand impersonation: Wise with suspicious attachment"
+description: |
+  Impersonating Wise Financial, as confirmed by NLU, and contains a suspicious attachment type.
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and (
+          strings.ilike(sender.display_name, '*wise*')
+          or strings.ilike(sender.email.domain.domain, '*wise*')
+      )
+  and any(ml.nlu_classifier(coalesce(body.html.display_text, body.plain.raw)).tags, .name in ("payment", "invoice"))
```

Comment:
@morriscode  we can update this to current thread

Reply:
ahh for sure!